apps/modernization-ui/src/confirmation/ConfirmationModal.tsx

```diff
@@ -5,6 +5,7 @@ import style from './confirmationModal.module.scss';
 
 type Props = {
     id?: string;
+    type?: string;
```

Comment:
The `ConfirmationModal` represents a simple Confirm/ Cancel operation.  Adding a `type` turns it into something else.  We may need to add a new type of modal to handle this type of interaction.